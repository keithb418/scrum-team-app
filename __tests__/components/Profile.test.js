import React from 'react';
import { shallow } from 'enzyme';
import Profile from '../../front-end/src/components/Profile';
import teamMemberData from '../fixtures/teamMembers';

test('should render the Profile component correctly', () => {
    const wrapper = shallow(<Profile member={teamMemberData[0]}/>);
    expect(wrapper).toMatchSnapshot();
});

test('should render the Profile component with an empty data', () => {
    const wrapper = shallow(<Profile member={{}}/>);
    expect(wrapper).toMatchSnapshot();
});