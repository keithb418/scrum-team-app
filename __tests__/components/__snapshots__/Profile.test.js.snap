// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the Profile component correctly 1`] = `
<Col
  bsClass="col"
  componentClass="div"
  md={6}
>
  <h3>
    Profile
  </h3>
  <ul
    className="member-info"
  >
    <li
      key="name"
    >
      Marc
    </li>
    <li
      key="email"
    >
      mtest@mail.com
    </li>
    <li
      key="teamLead"
    >
      Team Lead: 
      No
    </li>
    <li
      key="role"
    >
      Fullstack Developer
    </li>
    <li
      key="experience"
    >
      4 years
    </li>
  </ul>
</Col>
`;

exports[`should render the Profile component with an empty data 1`] = `
<Col
  bsClass="col"
  componentClass="div"
  md={6}
>
  <h3>
    Profile
  </h3>
  <ul
    className="member-info"
  />
</Col>
`;
